include/flatbuffers/idl.h

```diff
@@ -406,7 +410,7 @@ struct IDLOptions {
       generate_name_strings(false),
       escape_proto_identifiers(false),
       generate_object_based_api(false),
-      cpp_object_api_pointer_type("std::unique_ptr"),
+      cpp_object_api_pointer_type("flatbuffers::unique_ptr"),
```

Comment:
I think having non-STLPort users be able to use `std` types is pretty important, so yes, lets go this way.

I'd love to drop VS2010, since the pressure of people wanting to use more of C++11 is pretty great. This is something we can consider seperately. But first order of business is not requiring them use even less of C++11 because of STLPort :)